static/styles/left-sidebar.scss

```diff
@@ -17,6 +17,14 @@ $topic_indent: calc($far_left_gutter_size + $left_col_size + 4px);
     font-weight: 800;
 }
 
+/* Changes the color of left-sidebar scrollbar from the scrollbar used in
+rest of places */
+.simplebar-track {
```

Comment:
Hey, thanks for working on this. Did you check out the CSS related to simplebar in `static\styles\components.scss`?

```
.simplebar-track {
    .simplebar-scrollbar::before {
        background-color: hsl(0, 0%, 0%);
        box-shadow: 0px 0px 0px 1px hsla(0, 0%, 100%, 0.33);
    }
...
```